Update-AutomationAzureModulesForAccount.ps1

```diff
@@ -344,33 +401,33 @@ function Wait-AllModulesImported(
 # Uses the module import map created to import modules. 
 # It will only import modules from an element in the array if all the modules
 # from the previous element have been added.
-function Import-ModulesInAutomationAccordingToDependency([string[][]] $ModuleImportMapOrder) {
+function Import-ModulesInAutomationAccordingToDependency([string[][]] $ModuleImportMapOrder, [bool] $UseAzModule) {
 
     foreach($ModuleList in $ModuleImportMapOrder) {
         $i = 0
         Write-Output "Importing Array of modules : $ModuleList"
         foreach ($Module in $ModuleList) {
             Write-Verbose ("Importing module : {0}" -f $Module)
-            Import-AutomationModule -ModuleName $Module
+            Import-AutomationModule -ModuleName $Module -UseAzModule $UseAzModule
             $i++
             if ($i % $SimultaneousModuleImportJobCount -eq 0) {
                 # It takes some time for the modules to start getting imported.
                 # Sleep for sometime before making a query to see the status
                 Start-Sleep -Seconds 20
-                Wait-AllModulesImported $ModuleList $i
+                Wait-AllModulesImported $ModuleList $i $UseAzModule

```

Comment:
Please use explicit parameter names here and in line 423. (This should have been done even before, but now that we have 3 parameters the need for names is more visible.)